<launch>

    <arg name="scan_topic" default="/mur620a/UR10_r/line_laser/scan" />
    <arg name="center_point_topic" default="/profile/center_point" />
    <arg name="tf_target_frame" default="mur620a/base_link" />
    <arg name="lateral_offset_topic" default="/layer_center/offset_mm" />

    <!-- Node that runs the path transformer script -->
    <node name="find_profile_center" pkg="laser_scanner_tools" type="find_profile_center.py" output="screen">
        <!-- Transformation parameters -->
        <param name="scan_topic" value="$(arg scan_topic)" />
        <param name="center_point_topic" value="$(arg center_point_topic)" />
        <param name="tf_target_frame" value="$(arg tf_target_frame)" />
        <param name="lateral_offset_topic" value="$(arg lateral_offset_topic)" />

    </node>
</launch>

